content/docs/studio/user-guide/experiments/explore-ml-experiments.md

```diff
@@ -134,6 +134,17 @@ To reorder the columns, click and drag them in the table or from the Columns
 dropdown.
 ![Showing and hiding columns](https://static.iterative.ai/img/studio/reorder_columns.gif)
 
+**Columns menu and goals:** Click on the column header to open a context menu
+with actions such as sorting and filtering the project table by the column's
+values.
+
+For metrics, you can also specify goals, which indicate whether an increase or a
+decrease in the metric's value is desirable. Values better than that in the
+baseline row are highlighted in green, with the best one shown with a green
+border. Worse values are highlighted in pink.
```

Comment:
Better or worse than what? Is there some benchmark value that can be set? How does studio decide the threshold between green/pink?

Reply:
@dberenbaum It says `Values better than that in the baseline row are highlighted in green...`. Do you think this needs to be made clearer? How about this: `Once a goal is set, the metric's values for all rows are compared against the value in the baseline row. Values that are better (higher or lower, depending on the goal) than that in the baseline row are highlighted in green, with the best one shown with a green border. Values that are worse than that in the baseline row are marked in pink.`

Reply:
@dberenbaum I've revised the text [here](https://github.com/iterative/dvc.org/pull/4932/commits/7e359e246e151405951f4d052f337f01972280e6). PTAL.

Reply:
Do we need some explanation of the baseline row somewhere, the default value, and how to set it?

Reply:
We've use the term `baseline` in the docs elsewhere also without describing it. The project table displays the word `BASELINE` in the baseline row, so it should be clear from there. Still, I've now added an admon, with gif, explaining how to change the baseline row. ptal at [this commit](https://github.com/iterative/dvc.org/pull/4932/commits/a3270e60f150a3560a76e01929133bab8518dad6).

Reply:
@dberenbaum Let me know if you have more comments on this.